### DISCUSSION TITLE
[RFC]: modern.js 开发cms企业官网性能如何？与 hugo 相比的话

### DISCUSSION THREAD

**Discussion by moweiwei** [web-infra-dev/modern.js#3150] [General]
### 这个功能解决了什么问题？

modern.js 开发cms企业官网性能如何？与 hugo 相比的话

### 你期望的 API 是什么样子的？

 -

**Comment by chenjiahan** [comment]
目前没有和 hugo 做过性能对比呢。

Modern.js 提供了 SSR、SSG、流式渲染等优化手段，对于性能要求较高的场景，可以通过开启这些优化来达到更好的性能
